Misc/NEWS.d/next/Library/2024-02-09-07-20-16.gh-issue-115165.yfJLXA.rst

```diff
@@ -0,0 +1,3 @@
+``typing.Annotated`` ignores exceptions when setting the ``__orig_class__``
+attribute on objects returned from a callable. Previously only ``TypeError``
+was ignored.
```

Comment:
```suggestion
Ignore most exceptions when attempting to set the ``__orig_class__`` attribute
on objects returned when calling a generic alias. Previously only
:exc:`AttributeError` was ignored.
```

Also, feel free to give yourself credit if you like; it's common to do so:

```suggestion
Ignore most exceptions when attempting to set the ``__orig_class__`` attribute
on objects returned when calling a generic alias. Previously only
:exc:`AttributeError` was ignored. Patch by Dave Shawley.
```

Reply:
🤦 _Previously only TypeError_ ... good 👀

Reply:
addressed in 1d7e6e5b1a750793d09e5f8922da573c4a1b5c53